references.go

```diff
@@ -25,8 +25,6 @@ func newReferencesTable() sql.Table {
 	return new(referencesTable)
 }
 
-var _ Table = (*referencesTable)(nil)
```

Comment:
why did you remove this?

Reply:
It was a mistake, I thought it was `sql.Table` (which is already checked by `var _ sql.PusdownProjectionAndFiltersTable = (*referencesTable)(nil)` but this is checking for the gitbase table interace. I'll re-add that.